### PR TITLE
Improve docs for porter or duffle

### PR DESCRIPTION
**Pull request by carolynvs**
* Make it easier to find the page
* Refresh docs
* Summarize at the top and in the FAQ

Preview available at:
* https://deploy-preview-297--porter.netlify.com/faq/
* https://deploy-preview-297--porter.netlify.com/porter-or-duffle/

### CODE REVIEW

File: docs/content/porter-or-duffle.md [hunk 1]
```diff
@@ -137,7 +174,7 @@ The `porter build` command handles:
 So it's still there, but you don't have to mess with it. 😎 A few of the sections in the Porter manifest to map 1:1
 to sections in the bundle.json file, such as the bundle metadata, Parameters, and Credentials.
 
-## Hot Wiring a Bundle
+### Hot Wiring a Bundle
 I lied, they aren't actually 1:1 mappings. Porter has some special sauce, the `source` reference that makes it much
```
Comment (vdice) [inline docs/content/porter-or-duffle.md]: For a follow-up; `source` jumped out at me -- are there lingering docs that need updating per Porter's switch from `source` value interpolation to the current mustache templating approach?

File: docs/content/porter-or-duffle.md [hunk 2]
```diff
@@ -1,59 +1,95 @@
 ---
-title: Porter vs. Duffle
-description: Does Porter Replace Duffle?
+title: Porter or Duffle?
+description: A comparison of Porter, Duffle and when you would choose one over the other
 ---
 
-<p align="center">NOPE! 😁</p>
+<h5>A comparison of porter, duffle and when you would choose one over the other</h5>
```
Comment (vdice) [inline docs/content/porter-or-duffle.md]: nit: Should we capitalize Porter and Duffle here, for consistency?

File: docs/content/porter-or-duffle.md (hunk 1 above)
Comment (jeremyrickard) [reply docs/content/porter-or-duffle.md]: Yeah probably. I'm doing a doc scrub before DockerCon too.

Comment (carolynvs) [reply docs/content/porter-or-duffle.md]: Oops, yeah this should get updated in a follow-up. Thanks!

File: docs/content/porter-or-duffle.md (hunk 2 above)
Comment (carolynvs) [reply docs/content/porter-or-duffle.md]: Yes, thanks! I'll fix that before I merge

**Pull request merged**